### ISSUE TITLE
Deno support

### ISSUE THREAD

**Issue by oscarotero** [postcss/postcss-import#440]
Hi.
I've created this repository: https://github.com/oscarotero/postcss-deno-import
that converts the code of this plugin to be compatible with Deno.

It's similar to what I did to postcss main repo: https://github.com/postcss/postcss-deno

I couldn't execute all tests because the library used (`ava`) is difficult to replicate in Deno, but I could run some tests and seems to work fine in most cases (although I had to remove sugarSS support).

I like to know if you're interested in moving the repo to postcss organization and create a github action to sync the commits of this repo with the Deno version. Something similar to https://github.com/postcss/postcss/blob/master/.github/workflows/deno.yml

**Comment by RyanZim** [comment]
I don't know much of anything about deno (I have been meaning to look into it for months, but have not gotten time). I'm not opposed to adding support, but, like I said, I don't know much, and I have very little time to put into this. How much work was this to do for postcss? CC @ai

**Comment by ai** [comment]
@RyanZim Deno support is working by compiling sources and @oscarotero will support convertor.

I think it is pretty safe for you. You will need just to add GitHub Actions workflow.

I think it is cool to support Deno since we need more diversity in JS environments. I saw a sad moment in Ruby community history when they stuck with Ruby on Rails and they killed their future. I think supporting different enviornments is important to keep JS community development and innovations.

**Comment by RyanZim** [comment]
OK then, PR welcome. I will need to depend on @oscarotero to resolve any issues with the deno version; I am already stretched thin enough with postcss as it is, I cannot take on more responsibility.

**Comment by oscarotero** [comment]
Great!

- The Deno version is published here: https://deno.land/x/postcss_import (I've created the 0.1.0 tag to have an initial release). The name is `postcss_import`, with underscore, because dashes are not allowed)
- I've moved my repository to postcss organization: https://github.com/postcss/postcss-deno-import
- And, created a PR to add an action to this repo, in order to sync both repositories https://github.com/postcss/postcss-import/pull/441

If everything works fine, the next tag created (ex: `13.0.1`) will create the same tag in deno.
